src/swsssdk/configdb.py

```diff
@@ -93,10 +173,12 @@ def get_config(self):
         hashes = client.keys('*')
         data = {}
         for _hash in hashes:
-            table_name = _hash.split(':', 1)[0]
-            key = _hash.split(':', 1)[1]
-            if not data.has_key(table_name):
-                data[table_name] = {}
-            data[table_name][key] = client.hgetall(_hash)
+            tokens = _hash.split(':', 1)
+            if len(tokens) == 2:
+                table_name = _hash.split(':', 1)[0]
+                key = _hash.split(':', 1)[1]
+                if not data.has_key(table_name):
+                    data[table_name] = {}
+                data[table_name][key] = self.__raw_to_typed(client.hgetall(_hash))
```

Comment:
The entry of deleted key will be returned as None. Do you have any concern regarding that?

---
In reply to: [129326056](https://github.com/Azure/sonic-py-swsssdk/pull/8#discussion_r129326056) [](ancestors = 129326056)